<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>968</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(165, 165, 165);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="latteButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>309</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 81, 81);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ice Latte</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="imgAmericano" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="imgIcelatte" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>69</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="text">
     <string>* 이미지는 실제와 다를 수 있습니다.</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>729</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>LG Smart UI Regular</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>주문 내역</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>729</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>730</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="num1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>850</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="num2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>850</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="orderButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>730</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 81, 81);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>주문 하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 81, 81);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="num1_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1500원</string>
    </property>
   </widget>
   <widget class="QLabel" name="num2_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>2000원</string>
    </property>
   </widget>
   <widget class="QPushButton" name="milkButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>620</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 81, 81);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ice Milk</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="imgWater" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="waterButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 81, 81);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ice Water</string>
    </property>
   </widget>
   <widget class="QWidget" name="imgMilk" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="num2_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>581</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>800원</string>
    </property>
   </widget>
   <widget class="QLabel" name="num1_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>500원</string>
    </property>
   </widget>
   <widget class="QWidget" name="imgShot" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="num2_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>581</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>1000원</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shotButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>620</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 81, 81);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ice Shot</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="num3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>850</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>730</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>730</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="num4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>850</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_6" native="true">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>730</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="num5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>850</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>LG Smart UI Regular</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="americanoButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 81, 81);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ice Americano</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
